webplugin/css/app/style.css

```
@@ -64,7 +100,8 @@ textarea {
   border-radius: 6px;
   text-rendering: optimizeLegibility;
   -webkit-box-shadow: 0 1.5rem 2rem rgba(0, 0, 0, 0.3);
-  box-shadow: 0 1.5rem 2rem rgba(0, 0, 0, 0.3);
+  -webkit-box-shadow: var(--km-card-shadow);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

-webkit-box-shadow is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)